src/mumble/CMakeLists.txt

```diff
@@ -514,9 +514,9 @@ if(bundled-opus)
 		install(TARGETS opus LIBRARY DESTINATION lib)
 	endif()
 else()
-	find_pkg(Opus CONFIG REQUIRED)
-	add_dependencies(mumble PRIVATE Opus::opus)
-	target_include_directories(mumble PRIVATE ${Opus_INCLUDE_DIRS})
+	find_pkg(opus CONFIG REQUIRED)
```

Comment:
Here we might get away using something like
https://github.com/mumble-voip/mumble/blob/ae58ed8bc3e94d0469a353e24b6d298438a444de/src/mumble/CMakeLists.txt#L325
in order to find Opus regardless of the spelling. Though I guess this would mess with the name for the set variables. I think I'll have to check that.

Reply:
Looking at the code for `find_pkg` I don't think that using different spellings would work in this case...

On a different note though: Did you try not adding `CONFIG` as an argument here? In theory the function should automatically check for that so it shouldn't be necessary :thinking:

Reply:
It does work for me.

It's the same as when using `pkg-config` on my system.
`pkg-config --libs Opus` says that "Opus" doesn't exist.
If I do `pkg-config --libs opus` it works.

I observe the same thing with `find_pkg`:
 `Opus` -> Can't find library
`opus` -> finds the installed opus library

Just removing `CONFIG` does not help. I need to change it to lower case to work (at least on my system).

Reply:
Yeah the lowercase name is mandatory. I was only wondering if we need the `CONFIG`. However I just realized that it wasn't you who added the `CONFIG` keyword, so I assume it is indeed needed xD